result/bgpv6.yaml

```diff
@@ -170,4 +174,16 @@ components:
             openconfirm:
               x-field-uid: 5
             established:
-              x-field-uid: 6
\ No newline at end of file
+              x-field-uid: 6
+        gr_restart_attempted:
+          description: |-
+            Number of Graceful Restarts attempted by BGP Peer per session.
```

Comment:
For these , good to specify it is for peer graceful restarts probably when we are acting as Helper. ( assuming that is the case ) . Also good to specify exactly when they will be incremented. It is not very obvious to me.